### ISSUE TITLE
Hit Count Regulation is off in RAP64 0.57

### ISSUE THREAD

**Issue by kdecks** [GameDragon2k/RASuite#6]
The hitcounts appear to be slower than expected compared to previous versions, perhaps 1/3 the original speed.

If the memory view is opened, hitcounts count at approximately 10x their previous amount. Closing the mem viewed doesn't change anything. And an emulator restart is required to return to the previous slow speed.

**Comment by GameDragon2k** [comment]
Hmm, likely has something to do with the refreshing. I've tied the achievement data (that includes hit count) to update when the game refreshes/updates per frame. Should be more accurate.

Won't close this yet until I can confirm everything is working good, but seems like it from my end.

**Comment by GameDragon2k** [comment]
Haven't heard any issues, so going to assume this is fine.

**Comment by kdecks** [comment]
Did you attempt to make a fix for this it's a weird issue but Most
achievements don't use counters as a timer

On Feb 25, 2018 2:56 PM, "GameDragon2k" <notifications@github.com> wrote:

Haven't heard any issues, so going to assume this is fine.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/GameDragon2k/RASuite/issues/6#issuecomment-368348648>,
or mute the thread

**Comment by GameDragon2k** [comment]
What does that mean? "Most
achievements don't use counters as a timer"

**Comment by kdecks** [comment]
Sorry, was awkward typing from my phone.

As I reported earlier hit counting speeds were acting very irregular on Star Fox 64 during my tests. I reworked the achievement so it didn't rely on hitcounts as a timer.

Most achievements don't use number of hit counts as an accurate timer, but some do and all that might happen is something would trigger a little sooner than normal or later, but it might not be easy for the casual observer to see.
